src/prob/opf_capc.jl

```diff
@@ -0,0 +1,132 @@
+"Solve OPF with capacitor control"
```

Comment:
should correct to follow new conventions for docstring format that include the function signature